ensembl_production/templates/inc/footer.inc.html

```diff
@@ -14,7 +14,7 @@
             <div class="col-md-4"><span class="float-right">
                 <img src="{% static "img/logo.png" %}" alt="Ensembl Production" class="img-fluid"
                      style="max-width: 60px;">
-                Ensembl Production 2019</span>
+                Ensembl Production 2020</span>
```

Comment:
We could even think of making it.... dynamic :-)

Reply:
Done in 981632e